SwrveCommonSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -1848,4 +1831,30 @@ public C getConfig() {
         }
         return null;
     }
+
+    @Override
+    public List<SwrveConversationCampaign> getConversationCampaigns() {
+        List<SwrveConversationCampaign> result = new ArrayList<SwrveConversationCampaign>();
+        synchronized (campaigns) {
+            for (int i = 0; i < campaigns.size(); i++) {
+                SwrveBaseCampaign campaign = campaigns.get(i);
+                if (campaign instanceof SwrveConversationCampaign) {
+                    result.add((SwrveConversationCampaign)campaign);
+                }
+            }
+        }
+        return result;
+    }
+
+    @Override
+    public boolean displayConversationCampaign(SwrveConversationCampaign campaign) {
+        if (campaign != null && campaign.getConversation() != null && conversationListener != null) {
```

Comment:
I've made our SDK check if the campaigns are still active before returning them in the new method